chain/consensus/actors/atomic-replace/replace.go

```diff
@@ -0,0 +1,264 @@
+package replace
+
+// Sample actor that replaces the cid from one address to the other.
+// This actor is used as an example of how to use the actor execution
+// protocol.
+
+import (
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/cbor"
+	"github.com/filecoin-project/go-state-types/exitcode"
+	actor "github.com/filecoin-project/lotus/chain/consensus/actors"
+	"github.com/filecoin-project/lotus/chain/consensus/hierarchical/atomic"
+	"github.com/filecoin-project/specs-actors/v7/actors/builtin"
+	"github.com/filecoin-project/specs-actors/v7/actors/runtime"
+	cid "github.com/ipfs/go-cid"
```

Comment:
non-critical: unsorted

Reply:
In this case, as the package name is `/ipfs/go-cid` the linter suggests (it actually forces in my case) to be after `/filecoin-project/*` dependencies.

Reply:
sure. I meant grouping them into several groups as in other files.

```
         cid "github.com/ipfs/go-cid"

         "github.com/filecoin-project/lotus/chain/consensus/hierarchical/atomic"
	"github.com/filecoin-project/specs-actors/v7/actors/builtin"
	"github.com/filecoin-project/specs-actors/v7/actors/runtime"
	
```

Reply:
Oh, gotcha. Thanks for the clarification.